rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -312,6 +316,16 @@ def delete_price_for_nft(self, asset: Asset) -> bool:
 
         return True
 
+    def _filter_ignored_nfts(self, nfts_data: Dict[ChecksumEvmAddress, List[NFT]]) -> None:
+        """Remove ignored NFTs from NFTs data."""
+        with self.db.conn.read_ctx() as cursor:
+            ignored_nfts = {nft.identifier for nft in self.db.get_ignored_assets(cursor=cursor, only_nfts=True)}  # noqa: E501
```

Comment:
for how assets comparison work I believe you don't need this list comprehension. You can do

```python
ignored_nfts = self.db.get_ignored_assets(cursor=cursor, only_nfts=True)
```
and then 
```python
if nft.token_identifier in ignored_nfts
```